charts/victoria-metrics-single/values.yaml

```diff
@@ -372,6 +372,12 @@ server:
     # -- Service Monitor metricRelabelings
     metricRelabelings: []
 
+  # -- Global relabel configuration
+  relabel:
+    enabled: false
+    config: []
+    configMap: ""
```

Comment:
WDYT about either making this:

```
configMap:
  name: ""
  key: ""
```

Or adding a comment that key is expected to be a `relabel.yml` for this to work?

Reply:
done